master/internal/resourcemanagers/scheduler.go

```diff
@@ -13,15 +13,15 @@ type Scheduler interface {
 }
 
 // MakeScheduler returns the corresponding scheduler implementation.
-func MakeScheduler(schedulingPolicy string) Scheduler {
-	switch schedulingPolicy {
+func MakeScheduler(config *SchedulerConfig) Scheduler {
+	switch config.GetType() {
 	case priorityScheduling:
-		return NewRoundRobinScheduler()
+		return NewPriorityScheduler(config)
 	case fairShareScheduling:
 		return NewFairShareScheduler()
 	case roundRobinScheduling:
 		return NewRoundRobinScheduler()
```

Comment:
Are we going to make another case for round robin scheduler?

Reply:
Why do we need another case?

Reply:
I need to see an optometrist - we're good, for some reason I only saw two cases, not the round robin one.

Reply:
LMAO